frontend/src/libraries/explorer-wamp/blocks.ts

```diff
@@ -61,6 +61,13 @@ export default class BlocksApi extends ExplorerApi {
           paginationIndexer,
         },
       ]);
+      const finalHeight = await this.getFinalStats();
```

Comment:
This naming is really misleading, and you can notice it right at this line, you save "final *stats*" to "final *height*". Is height the only final stat? Either function, or extract the finalHeight from the final block header.